python/nvstrings.py

```diff
@@ -339,6 +339,18 @@ def get_cpointer(self):
         """
         return self.m_cptr
 
+    def create_from_ipc(self):
+        """
+        Returns a valid NVStrings object from IPC data.
+        """
+        return pyniNVStrings.n_createFromIPC(self.m_cptr)
+
+    def get_ipc_data(self):
```

Comment:
Please do not mention internal, private, C++ class NVStringsImpl in the description.
Also, provide an example on using this method.

Reply:
Got it.